walk-in-interview/src/main/java/com/google/job/data/Job.java

```diff
@@ -14,7 +14,7 @@
     private final Location jobLocation;
     private final String jobDescription;
     private final JobPayment jobPay;
-    private final List<String> requirements;
+    private final Map<String, Boolean> requirements; // requirement stable id : true/false
```

Comment:
If a key is mapped to false, does the key need to be there?

Reply:
since we need to find those post that does not contain skills (mapped to false) that applicant does not have (negate skills), so we need to map it to false; else cloud firestore cannot do "not in"